maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/dto/maat_api/ApplicantDTO.java

```diff
@@ -15,7 +15,12 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class ApplicantDTO implements Serializable {
-    @JsonValue
+
+    @JsonCreator(mode = JsonCreator.Mode.DELEGATING)
+    public ApplicantDTO(int id) {
```

Comment:
Could you please add a unit test for the deserialisation passing the json?

Reply:
Done